api/xverse.ts

```diff
@@ -92,6 +92,34 @@ export async function getCoinsInfo(
     });
 }
 
+/**
+ * get BRC-20 supported tokens with the fiat rate
+ * @param network
+ * @param tickers provided to get the fiat rate along with supported tokens
+ * @param fiatCurrency
+ */
+export async function getBrc20Tokens(
+  network: NetworkType,
+  tickers: string[],
+  fiatCurrency: string,
+): Promise<CoinsResponse | null> {
+  const url = `${XVERSE_API_BASE_URL(network)}/v1/brc20/tokens`;
+
+  const params = {
+    currency: fiatCurrency,
+    tickers: tickers,
+  };
+
+  return axios
+    .get<CoinsResponse>(url, { params })
```

Comment:
this won't be CoinsResponse. see the type from xverse-api repo

Reply:
need to extend BaseToken as created from jordan's PR https://github.com/secretkeylabs/xverse-core/pull/353

Reply:
the BaseToken has `image` that we don't have in this API's response. I've added the response type

Reply:
think it's OK since image is optional on BaseToken